extensions/binding/org.eclipse.smarthome.binding.meteoblue/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,421 @@
+		<item-type>String</item-type>
+		<label>Cardinal Wind Direction</label>
+		<state readOnly="true" pattern="%s">
+		</state>
+	</channel-type>
+
+	<channel-type id="windDirection" advanced="true">
+		<item-type>Number</item-type>
+		<label>Wind direction</label>
+		<description>Wind direction (in degrees)</description>
+		<category>Wind</category>
+		<state readOnly="true" min="0" max="360" pattern="%.0f °">
+		</state>
+	</channel-type>
+
+	<channel-type id="maxWindSpeed">
+		<item-type>Number:Speed</item-type>
+		<label>Maximum Wind Speed</label>
+		<description>Maximum wind speed</description>
+		<category>Wind</category>
+		<state readOnly="true" pattern="%.1f %unit%" />
+	</channel-type>
+
+	<channel-type id="minWindSpeed">
+		<item-type>Number:Speed</item-type>
+		<label>Minimum Wind Speed</label>
+		<category>Wind</category>
+		<state readOnly="true" pattern="%.1f %unit%" />
+	</channel-type>
+
+	<channel-type id="meanWindSpeed">
+		<item-type>Number:Speed</item-type>
+		<label>Mean Wind Speed</label>
+		<category>Wind</category>
+		<state readOnly="true" pattern="%.1f %unit%" />
+	</channel-type>
+
+	<channel-type id="pressureMin">
+		<item-type>Number:Pressure</item-type>
+		<label>Low Pressure</label>
+		<description>Low Pressure</description>
+		<category>Pressure</category>
+		<state readOnly="true" pattern="%.2f %unit%" />
+	</channel-type>
+
+	<channel-type id="pressureMax">
+		<item-type>Number:Pressure</item-type>
+		<label>High Pressure</label>
+		<description>High Pressure</description>
+		<category>Pressure</category>
+		<state readOnly="true" pattern="%.2f %unit%" />
+	</channel-type>
+
+	<channel-type id="pressureMean">
+		<item-type>Number:Pressure</item-type>
+		<label>Mean Pressure</label>
+		<description>Mean Pressure</description>
+		<category>Pressure</category>
+		<state readOnly="true" pattern="%.2f %unit%" />
+	</channel-type>
+
+	<channel-type id="UVIndex" advanced="true">
+		<item-type>Number</item-type>
+		<label>UV Index</label>
+		<description>UV Index</description>
+		<state readOnly="true" pattern="%.1f">
+		</state>
+	</channel-type>
+
+	<channel-type id="precipitation">
+		<item-type>Number:Length</item-type>
+		<label>Precipitation</label>
+		<description>Prediction of total precipitation</description>
+		<category>Rain</category>
+		<state readOnly="true" pattern="%.2f %unit%" />
+	</channel-type>
+
+	<channel-type id="convectivePrecipitation">
+		<item-type>Number:Length</item-type>
+		<label>Convective Precipitation</label>
+		<description>Prediction of convective precipitation (rainfall)</description>
+		<category>Rain</category>
+		<state readOnly="true" pattern="%.2f %unit%" />
+	</channel-type>
+
+	<channel-type id="rainSpot">
+		<item-type>String</item-type>
+		<label>Rain Spot</label>
+		<description>Precipitation distribution around the location as a 7x7 array,
+		encoded from south to north and west to east
+		(e.g. 2112211093222219212110112229922133311912239022192).</description>
```

Comment:
Because this is raw technical data, which does not make any sense for a user to e.g. display in a UI. For this, we have the Image channel, which makes sense. We must not add channels just because of the sake of having some random data at hand.